mysteries-of-gods-kingdom-declared/updated/02-to-the-reader.adoc

```diff
@@ -43,3 +43,5 @@ Francis Howgill
 
 [.signed-section-context-close]
 1658
+
+//Does this whole section need a embedded document thing?
```

Comment:
Nope, this is an example of one that does NOT need to be embedded, because, even though he signed it at the end, its not really a letter. Also, as I said in another review I just did, the whole purpose for the embedded section is purely visual... it is so that the reader can glance at the page and say, "OH... here starts a letter! Or here is where the letter ends! I can tell because it is indented, uses a different font and spacing, has an indented greeting and signature, etc."  So you can just remove your comment here, and we'll merge this one as is.